netutils/tinydtls/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# ##############################################################################
+# apps/netutils/tinydtls/CMakeLists.txt
```

Comment:
should we move tinydtls to crypto folder?

Reply:
true, it fits better in crypto along with mbedtls and wolfssl

Reply:
done